gulpfile.js

```diff
@@ -1092,7 +1092,7 @@ function createCesiumJs() {
 define([' + moduleIds.join(', ') + '], function(' + parameters.join(', ') + ') {\n\
   \'use strict\';\n\
   var Cesium = {\n\
-    VERSION : "' + version + '",\n\
+    VERSION : \'' + version + '\',\n\
```

Comment:
This is the one by-hand change I had to make. Otherwise ESLint fails when checking `Source/Cesium.js`